generators/app/templates/src/app/components/Routes/paths.ts

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
I'd move this to the constants file since it's relevant to the routes definition.

Reply:
I had to move it elsewhere because it was generating circular dependencies. Routes constant import components and those components sometimes import paths. If those paths are in Routes, that's a circular dependency. Given that components shouldn't import routes, only path, I think it might be actually better this way.